site/dlx.xml

```diff
@@ -230,9 +230,10 @@ args.put("x-dead-letter-routing-key", "some-routing-key");</pre>
             expiring again in any queues it is routed to.</li>
         </ul>
 
-        In case <code>x-death</code> already contains an entry with
+        New entries are prepended to the beginning of the <code>x-death</code>
+        array. In case <code>x-death</code> already contains an entry with
         the same queue and dead lettering reason, its count field will be
-        incremented.
+        incremented and it will be moved to the beginning of the array.
```

Comment:
Are they not? In https://github.com/rabbitmq/rabbitmq-server/issues/78 @simonmacmullen proposed this (emphasis mine):

> Look for a matching one, matching on both queue and reason
> If there, increment its count **and bring to the front of the list**
> If not, create one with count=1 and add to the front of the list

My Erlang isn't so great so I couldn't immediately spot if this was actually happening in the related commit, but assumed this is how it works now. It would be nice because it would make determining the last dead letter reason as easy as grabbing the first entry from x-death. If this is not the case, I will change the phrasing to make this clear.

So to be clear, in this scenario:
```
- rejected on queue foo at 00:00
- rejected on queue bar at 00:01
- rejected on queue foo at 00:02
- rejected on queue baz at 00:03
```
the order in x-death would be
```
- baz, 1x, last time at 00:03
- bar, 1x, last time at 00:01
- foo, 2x, last time at 00:02
```
?